packages/angular_devkit/build_angular/src/differential-loading/update-webpack-config.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+type WebpackConfig = any; // tslint:disable-line
```

Comment:
why? either give a local type that matches the shape, or import a correct type, or explain in the //tslint comment why you can't